### DISCUSSION TITLE
Remove gradle push task

### DISCUSSION THREAD

**Discussion by matinzd** [lottie-react-native/lottie-react-native#894] [General]
Hi guys,
Do we even need this task to be applied to the main project? I think it's better to remove it.

https://github.com/lottie-react-native/lottie-react-native/blob/ca013acf8d0a2ef76463baf59212daa14821635d/src/android/build.gradle#L6

Causing this issue: https://github.com/lottie-react-native/lottie-react-native/issues/919

**Comment by matinzd** [comment]
@emilioicai @AlexanderEggers

**Comment by AlexanderEggers** [comment]
Yeah agree, let's do that when the android fabric work lands 👍

**Comment by emilioicai** [comment]
Removing sounds good to me

**Comment by matinzd** [comment]
Removed in https://github.com/lottie-react-native/lottie-react-native/commit/5bff05c159852ecd06d9a0206dc525c18e8a07f9.
